src/component/metric/index.tsx

```diff
@@ -35,8 +39,42 @@ function Metric(props: MetricProps) {
   // 4. Props to set on the root node.
   //
   // By default, we are just piping all MetricProps here, but feel free
-  // to do whatever works for you.
-  return <PlasmicMetric {...props} />;
+  // to do whatever works for you. 
+  return <PlasmicMetric
+    name={'Customer Acquisition Cost'}
+    graphContainer={{
+      render: () => <LineChart dataKey={'cac'} data={[
```

Comment:
The import change should then result into something like this. 

```suggestion
      render: () => <linechart.Component dataKey={'cac'} data={[
```